FBSnapshotTestCase/FBSnapshotTestController.h

```diff
@@ -67,6 +67,11 @@ extern NSString *const FBDiffedImageKey;
 @property (readwrite, nonatomic, assign) BOOL usesDrawViewHierarchyInRect;
 
 /**
+ Adds view to UIWindow before capturing the snapshot
```

Comment:
Missing the period at the end (also for usesDrawViewHierarchyInRect above), and more importantly the description of what happens if the flag is NO (which I assume is the default).